src/main/java/app/gpx_animator/ui/GeneralSettingsPanel.java

```diff
@@ -824,13 +947,73 @@ protected Type configure(final JFileChooser outputFileChooser) {
         gbcSpeedUnit.gridx = 1;
         add(speedUnitComboBox, gbcSpeedUnit);
 
+
     }
 
     private void setVideoSize(final int width, final int height) {
         widthSpinner.setValue(width);
         heightSpinner.setValue(height);
     }
 
+
+    private List<MapTemplate> readMaps() {
+        final SAXParserFactory factory = SAXParserFactory.newInstance();
+        final SAXParser saxParser;
+        try {
+            saxParser = factory.newSAXParser();
+        } catch (final ParserConfigurationException | SAXException e) {
+            throw new RuntimeException(e);
+        }
+
+        final List<MapTemplate> labeledItems = new ArrayList<>();
+
+        try {
+            try (InputStream is = getClass().getResourceAsStream("/maps.xml")) { //NON-NLS
+                saxParser.parse(is, new DefaultHandler() {
+                    private final StringBuilder sb = new StringBuilder();
+                    private String name;
+                    private String url;
+                    private String attributionText;
+
+                    @Override
+                    @SuppressWarnings("checkstyle:MissingSwitchDefault") // Every other case can be ignored!
+                    @SuppressFBWarnings(value = "SF_SWITCH_NO_DEFAULT", justification = "Every other case can be ignored!") //NON-NLS NON-NLS
+                    public void endElement(final String uri, final String localName, @NonNls final String qName) {
+                        switch (qName) {
+                            case "name":
+                                name = sb.toString().trim();
+                                break;
+                            case "url":
+                                url = sb.toString().trim();
+                                break;
+                            case "attribution-text":
+                                attributionText = sb.toString().trim();
+                                break;
+                            case "entry":
+                                labeledItems.add(new MapTemplate(name, url, attributionText));
+                                break;
+                        }
+                        sb.setLength(0);
+                    }
+
+                    @Override
+                    public void characters(final char[] ch, final int start, final int length) {
+                        sb.append(ch, start, length);
+                    }
+                });
+            } catch (final SAXException e) {
+                throw new RuntimeException(e);
+            }
+        } catch (final IOException e) {
+            throw new RuntimeException(e);
+        }
+
+        labeledItems.sort(Comparator.comparing(MapTemplate::toString));
+
+        return labeledItems;
+    }
+
+
```

Comment:
I moved this method to the `MapUtil` class three days ago. Just don't add it back to the `GeneralSettingsPanel` class. Remove it, and it will be fine.